### DISCUSSION TITLE
use the `latest` tag for the latest stable version, and `testing` for testing versions.

### DISCUSSION THREAD

**Discussion by FarisZR** [simple-login/app#466] [Self-hosting question] [answered]
with each release you have to change the tag in docker/docker-compose,

i would like to see simplelogin use the `latest` tag for the latest stable release
and `testing` or `git` for latest testing release.

would make updating much easier.

**Comment by nguyenkims** [comment, marked as answer]
@fareszr hey there's `simplelogin/app-ci` image which is automatically built for every change on `master` and is more up-to-date than the `simplelogin/app` image which is only released from time to time.

**Reply by FarisZR** [reply]
the issue is not with development versions, its with stable versions
using the latest tag for stable releases, will remove the need manually change tags for every update.

also for any one who wants a specific version thy can just specify which version in docker tags.

**Reply by nguyenkims** [reply]
Oh I think it's actually better to use an explicit version as upgrading between 2 `latest` might break the existing installation, for example when there's a breaking change.

**Reply by FarisZR** [reply]
> Oh I think it's actually better to use an explicit version as upgrading between 2 `latest` might break the existing installation, for example when there's a breaking change.

specific release tags will still exist, im just saying latest should be used for people who want semi-auto stable updates.

**Reply by nguyenkims** [reply]
Actually the problem with "semi-auto stable updates" is an update can be a breaking change. Let's say you're on the v2 and SimpleLogin releases the v3, set it to `latest`. Now when you try to to auto update, the code will break. 

That being said, it makes sense to create Docker tag like `simplelogin/app:v2` so if you're on v2.1.0, you can have auto update when if v2.5.1 is released. We'll try to add that in the next release then :)

**Reply by FarisZR** [reply]
yup that is the best solution
